presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -171,6 +171,7 @@
     public static final String REMOTE_FUNCTIONS_ENABLED = "remote_functions_enabled";
     public static final String CHECK_ACCESS_CONTROL_ON_UTILIZED_COLUMNS_ONLY = "check_access_control_on_utilized_columns_only";
     public static final String SKIP_REDUNDANT_SORT = "skip_redundant_sort";
+    public static final String ALLOW_WINDOW_ORDER_BY_ORDINALS = "allow_window_order_by_ordinals";
```

Comment:
Do you want to call it ordinals or literals? Just pick one! :P

Reply:
Oh yeah - I think literals is more appropriate as that's what we are doing. Fixed.